### PR TITLE
Update Linktree icon

### LINKED ISSUE #7955: Linktree icon out of date
### Brand Name

Linktree

### Official Resources for Icon and Color

https://linktr.ee/

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by Poukame**
linktree

**Issue:** closes #7955 

**Similarweb rank:**
[#432000](https://www.similarweb.com/website/linktree.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Got the new hex color from their favicon.

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
Hi ,

Thanks for your contribution to simple-icons!

I see some differences:
image

I used the header icon from the website. Maybe check for the precision?

**Comment by Poukame** [comment]
Hello @dirien,

I've refined it slighly. Which icon did you compare to? The tree alone or logo with text?
In my process, I used svg-grabber on their website and took the tree logo alone.

When I diff with this one, I'm near 100% match.

Hope it's gonna be ok.

**Review by dirien — APPROVED** [review]
Hi @Poukame ,

Thanks for your contribution to simple-icons!

It's a perfect fit:
image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6420,7 +6420,7 @@
         },
         {
             "title": "Linktree",
-            "hex": "39E09B",
+            "hex": "43E55E",
             "source": "https://linktr.ee/"
```
Comment (LitoMore) [inline _data/simple-icons.json:6424]:
```suggestion
            "source": "https://linktr.ee"
```

**Pull request merged**